config/deploy/docker/lib/deployer.rb

```diff
@@ -261,6 +263,8 @@ def _pre_cache_version
 
   def _set_image_tag!
     if ENV['IMAGE_TAG']
+      # TODO this overrides the specified revision/version - when is this used?
+      # seem like this would've already caused a mismatch with _set_revision
       self.image_tag = ENV['IMAGE_TAG']
       self.image_tag_latest = 'latest-' + ENV['IMAGE_TAG']
```

Comment:
open question here

Reply:
It's just for debugging and not part of normal deployments.

Reply:
okay great, I updated the comment to say that